src/workerd/io/trace.c++

```diff
@@ -581,8 +96,10 @@ kj::Own<WorkerTracer> PipelineTracer::makeWorkerTracer(PipelineLogLevel pipeline
     kj::Maybe<kj::String> dispatchNamespace,
     kj::Array<kj::String> scriptTags,
     kj::Maybe<kj::String> entrypoint) {
-  auto trace = kj::refcounted<Trace>(kj::mv(stableId), kj::mv(scriptName), kj::mv(scriptVersion),
-      kj::mv(dispatchNamespace), kj::mv(scriptId), kj::mv(scriptTags), kj::mv(entrypoint));
+  // TODO(streaming-trace): Pass the account id
```

Comment:
Is this TODO for now or a future PR?

Reply:
Yes ;-) ... both. It's not needed for this initial PR but it'll be handled by the time I'm done with this (however many PRs it ends up being)